src/main/xerus/monstercat/api/APIConnection.kt

```diff
@@ -74,23 +74,50 @@ class APIConnection(vararg path: String) : HTTPQuery<APIConnection>() {
 	fun getTracks() =
 		parseJSON(TrackResponse::class.java)?.results
 	
+	fun postLogin(username: String, password: String) =
```

Comment:
Please make this a static method. If you hardcode the JSON you might as well hardcode the URL

Reply:
Refactored in a81ff488b772515fd42a35feb308eb07e3cb503e